### ISSUE TITLE
[Enhancement] Allow to programmatically enable/disable the Tags dropdown element

### ISSUE THREAD

**Issue by jestrella04** [lekoala/bootstrap5-tags#26]
It would be nice to programmatically enable/disable the Tags dropdown element to deal with situations where tag selection have a dependency on information required to be entered/selected on another element.

**Comment by lekoala** [comment]
Fair point. Added in demo and available in https://github.com/lekoala/bootstrap5-tags/releases/tag/1.4.7
